plugins/experimental/stek_share/common.h

```diff
@@ -0,0 +1,67 @@
+/** @file
+
+  commmon.h - Things that need to be everywhere
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+ */
+
+#pragma once
+
+#include <iostream>
+#include <iomanip>
+#include <string>
+#include <cstring>
+#include <mutex>
+#include <deque>
+#include <cmath>
+
+#define PLUGIN "stek_share"
+
+class PluginThreads
```

Comment:
Is the PluginThreads class used?

Reply:
Yeah, this is for shutting down threads.

Reply:
Oh nevermind.  I didn't have all the files opened up.  I see its use now.